llvm/lib/Target/AMDGPU/SIISelLowering.cpp

```diff
@@ -5575,10 +5552,8 @@ MachineBasicBlock *SITargetLowering::EmitInstrWithCustomInserter(
     MachineBasicBlock *SplitBB = BB->splitAt(MI, false /*UpdateLiveIns*/);
     MachineBasicBlock *TrapBB = MF->CreateMachineBasicBlock();
     MF->push_back(TrapBB);
-    BuildMI(*TrapBB, TrapBB->end(), DL, TII->get(AMDGPU::S_ENDPGM))
-      .addImm(0);
-    BuildMI(*BB, &MI, DL, TII->get(AMDGPU::S_CBRANCH_EXECNZ))
-      .addMBB(TrapBB);
+    BuildMI(*TrapBB, TrapBB->end(), DL, TII->get(AMDGPU::S_ENDPGM)).addImm(0);
+    BuildMI(*BB, &MI, DL, TII->get(AMDGPU::S_CBRANCH_EXECNZ)).addMBB(TrapBB);
```

Comment:
clang-format always mangles BuildMI calls. I think all of these should be formatted with one add per line

Reply:
I don't think `clang-format` understands what the code "actually" means.

Reply:
Yes, it doesn't

Reply:
How do you suggest moving forward?

Reply:
Undo the cases where clang-format is wrong?

Reply:
In order to make future life easier, I guarded all of them with `// clang-format off`.